File: C/PRAK602-2210817310017-Rohid_Akmal.c
#include <stdio.h>

void main(){
    int b, i, in;
    scanf("%d", &b);
    int zetsu[b];
    for (i = 0; i < b; i++){
        scanf("%d", &in);
        zetsu[i] = in;
    }
    for(i = 0; i < b; i++){
        printf("%d ", zetsu[i] * (i + 1));
    }
}

File: Python/PRAK602-2210817310017-Rohid Akmal.py
k = int(input())
angka = list(map(int, input().split()))
zetsu = []
a = 0
for i in range(0, k) :
    zetsu.append(angka[a : a + k])
for i in range(0, k) :
    print(zetsu[0][i] * (i + 1), end=' ')